src/Symfony/Bundle/TwigBundle/Tests/Functional/AttributeExtensionTest.php

```diff
@@ -43,6 +43,7 @@ public function registerBundles(): iterable
             public function registerContainerConfiguration(LoaderInterface $loader): void
             {
                 $loader->load(static function (ContainerBuilder $container) {
+                    $container->setParameter('kernel.secret', 'secret');
```

Comment:
The lowest installed Twig version that is now installed is 3.21. This means that this test is not skipped anymore, but since FrameworkBundle 6.4 is installed we also need the `kernel.secret` parameter to be set as the `uri_signer` is not lazy (this was changed with #59086 in Symfony 7.2).